.editorconfig

```diff
@@ -1,10 +1,10 @@
 root = true
 
+[*.py]
+indent_size = 4
+
 [*]
 indent_style = space
-indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
-[*.yml]
-indent_size = 2
```

Comment:
**question**: isn't this useful for the ci parts of this project?

Reply:
I don't think that it's needed. VSCode automatically detects that they are two spaces.